pkg/stanza/fileconsumer/attrs/ownerInfo_other.go

```diff
@@ -0,0 +1,12 @@
+//go:build !linux && !solaris
```

Comment:
```suggestion
//go:build windows
```

And the name should be `owner_windows.go`